CHANGELOG

```diff
@@ -1,3 +1,8 @@
+Master
+------
+
+* Move SessionStorage from a generated class to a re-usable module
```

Comment:
good point - the migration path is pretty simple delete the old SessionStorage class and include the mixin